.changes/tinymce/7.7.0.md

```diff
@@ -0,0 +1,27 @@
+## 7.7.0 - 2025-02-17
+
+### Added
+- `link_attributes_postprocess` option that allows overriding attributes of a link that would be inserted through the link dialog. #TINY-11707
+
+### Improved
+- Improved visual indication of a keyboard focus for a comment annotation when there is an image inside. #TINY-11596
+- Not specifying a notification type (or specifying incorrect one) now defaults to 'info'. #TINY-11661
```

Comment:
```suggestion
- Not specifying a notification type (or specifying an incorrect one) now defaults to 'info'. #TINY-11661
```

Reply:
```suggestion
- `editor.notificationManager.open()` now has a default type of `info`. #TINY-11661
```

Reply:
I made a different variant. I don't like when the changelog items start with backticks looks weird.